test/unit/server/rpc/procedures/hawaii-co2-data.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('hawaii-co2-data', function() {
+    const utils = require('../../../../assets/utils');
+    var HawaiiCO2Service = utils.reqSrc('rpc/procedures/hawaii-co2-data/hawaii-co2-data'),
+        RPCMock = require('../../../../assets/mock-rpc'),
+        co2 = new RPCMock(HawaiiCO2Service);
+
+    utils.verifyRPCInterfaces(co2, [
```

Comment:
This has also been updated to ensure the final interface is being tested in https://github.com/NetsBlox/NetsBlox/pull/2472 and this has made some minor changes to writing these tests (it removes some of the extra boilerplate). An example showing how to update the test can be found [here](https://github.com/NetsBlox/NetsBlox/pull/2472/files#diff-001ac5464bf51dc96590e9dacf71c35fL1-L10).